### ISSUE TITLE
Integrate CLI services describe function into Landing page

### ISSUE THREAD

**Issue by davco01a** [kabanero-io/kabanero-landing#242]
#### Feature description
Integrate CLI services describe function into Landing page

#### Feature design or visual mockup
The describe function provides more detailed metadata on a given stack for informational and debug purposes

#### Additional context

endpoint:  https://{{myurl}}/v1/describe/stacks/nodejs/versions/0.3.6

JSON returned:
{
    “image”: “docker.io/kabanerobeta/nodejs-express”,
    “name”: “nodejs-express”,
    “project”: “kabanero”,
    “digest check”: “matched”,
    “version”: “0.2.10",
    “git repo url”: “https://raw.githubusercontent.com/kabanero-io/kabanero-command-line-services/master/index.yaml”,
    “image digest”: “17fd40a884f44345fa05b395c10c5cf48be7e92a3e21800c5d61b7b87724002e”,
    “kabanero digest”: “17fd40a884f44345fa05b395c10c5cf48be7e92a3e21800c5d61b7b87724002e”,
    “status”: “active”
    "applications":[].   // <--- work in progress on applications using this stacks
}

**Comment by davco01a** [comment]
could the landing page business logic gen the the kAppNav link to see more detailed data on the appsody app, so that Champ could just click on a link to see more app metadata

**Comment by alohr51** [comment]
yes we do that today if kAppNav is installed

**Comment by davco01a** [comment]
nice

**Comment by davco01a** [comment]
A lot of the metadata for the application is returned in the map, so it's possible I could just pass back a jsonobject that contains whatever we need and you can bypass having to use kAppNav

**Comment by davco01a** [comment]
[deploymentMap.log](https://github.com/kabanero-io/kabanero-landing/files/4615692/deploymentMap.log)

**Comment by davco01a** [comment]
e.g.

```
"applications": "[{"name":"nodejs","namespace":"kabanero","architecture":"x86_64"}]"

**Comment by alohr51** [comment]
yeah that sounds good, you can just return the whole thing (if it isn't excessively large).

**Comment by davco01a** [comment]
take a look at deploymentMap.log above, the map is sort of big

**Comment by davco01a** [comment]
ok, got the map into the json response:

```
"applications": "[{\"name\":\"nodejs\",\"namespace\":\"kabanero\",\"map\":{\"metadata\":.....

**Comment by davco01a** [comment]
[applications1.log](https://github.com/kabanero-io/kabanero-landing/files/4629462/applications1.log)

**Comment by davco01a** [comment]
json response from new GO Kababenero REST service:

{
    "apps": [
        {
            "app.kubernetes.io/instance": "nodejs",
            "app.kubernetes.io/managed-by": "appsody-operator",
            "app.kubernetes.io/name": "nodejs",
            "app.kubernetes.io/part-of": "nodejs",
            "app.kubernetes.io/version": "1.0.0"
        },
        {
            "app.kubernetes.io/instance": "nodejs",
            "app.kubernetes.io/managed-by": "appsody-operator",
            "app.kubernetes.io/name": "nodejs",
            "app.kubernetes.io/part-of": "nodejs",
            "app.kubernetes.io/version": "1.0.0"
        }
    ],
    "digest check": "matched",
    "image digest": "f13c6c3022544b6ffc4f0f111b36c0f4d0066accb3d9e9a7f7d1218a7411c159",
    "image name": "docker.io/kabanero/nodejs",
    "kabanero digest": "f13c6c3022544b6ffc4f0f111b36c0f4d0066accb3d9e9a7f7d1218a7411c159",
    "name": "nodejs",
    "project": "kabanero",
    "status": "active",
    "version": "0.3.6"
}

**Comment by davco01a** [comment]
Old CLI service does not provide list of apps in describe response

New GO service does
